src/components/atoms/Ranges/LevelRange/LevelRange.styled.ts

```diff
@@ -0,0 +1,167 @@
+import styled from 'styled-components';
+import colors from 'styles/colors';
+
+export const StyledWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100%;
+`;
+export const Label = styled.label`
+  position: relative;
+`;
+
+export const ThumbWrapper = styled.div<{ value: number }>`
+  height: 24px;
+  width: 24px;
+  position: absolute;
+  bottom: -6px;
+  left: ${({ value }) => `${value}%`};
+  transform: translateX(-50%);
+  z-index: 2;
+`;
+
+export const ThumbSlider = styled.span<{ value: number }>`
+  position: absolute;
+  top: 0;
+  width: 24px;
+  height: 24px;
+  background: ${colors.background.NeutralWhite};
+  border-radius: 50%;
+  border: 2px solid ${colors.background.Primary20};
+  cursor: pointer;
+  ::before {
+    content: '${({ value }) => value}';
+    position: absolute;
+    top: -25px;
+    display: block;
+    text-align: center;
+    width: 100%;
+    font-family: Lato;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 17px;
+    line-height: 135%;
+    cursor: default;
+    text-align: center;
+    color: #0d54a5;
```

Comment:
use color from global styles

Reply:
This one missing but #4694f0 we don't have in our colors.